sound/soc/sof/topology.c

```diff
@@ -2541,7 +2544,7 @@ static int sof_dai_load(struct snd_soc_component *scomp, int index,
 		return ret;
 
 	dev_vdbg(scomp->dev, "tplg: pcm %s stream tokens: capture d0i3:%d\n",
-		 spcm->pcm.pcm_name, spcm->stream[1].d0i3_compatible);
+		 spcm->pcm.pcm_name, spcm->stream[stream].d0i3_compatible);
 
```

Comment:
@lyakh stream = SNDRV_PCM_STREAM_CAPTURE ?

Reply:
@kv2019i this is done on line 2540

Reply:
Aa, my bad, then it's ok.